deployable/nodejs/app.js

```diff
@@ -121,12 +121,6 @@ function triggerTest(message) {
       ? Buffer.from(message.data, 'base64').toString()
       : console.error("WARNING: no log function was invoked");
 
-  console.log('Fn invoked with attributes, if any: ');
-  console.log(message.attributes);
-
-  if (message.attributes) {
-    tests[testName](message.attributes['log_name'], message.attributes['log_text']);
-  } else {
-    tests[testName]();
-  }
+  args = message.attributes ? message.attributes : {};
```

Comment:
i am not familiar with the logic here. but the fix may result in submitting more attributes than just the following two: `"log_name"` and `"log_text"`. is it the intended behavior?

Reply:
Yes, that's the intention. We want to be able to trigger test functions with arbitrary arguments (I added severity here already, and Python makes use of others)